bundle/manifests/runtime-component.clusterserviceversion.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
             "name": "runtimecomponent-sample"
           },
           "spec": {
-            "applicationImage": "icr.io/appcafe/open-liberty/samples/getting-started@sha256:9a6f6352bee814302fcf939dd640b7d3b3e02307330f82bc32b782e7a9feb8a3",
+            "applicationImage": "icr.io/appcafe/open-liberty/samples/getting-started@sha256:e22dd56a05e44618a10d275d3ff07a38eb364c0f04f86ffe9618d83dd5467860",
             "expose": true,
             "manageTLS": true,
             "replicas": 1,
```

Comment:
The sample for v1beta2 need to be updated as well. The update-sample.sh missed it.